playbook.yml

```diff
@@ -70,14 +70,14 @@
           name: "load-ycsb"
         vars:
           ycsb: "{{ workload.args }}"
-        when: workload.name == "ycsb" and workload.args.workloads | length > 0
+        when: workload.name == "ycsb" and workload.args.workloads | length > 0 and load_ycsb
```

Comment:
does this really need to be a variable defined in group_vars/all.yml? Feels like it could just be 
```workload.name == "ycsb" and workload.args.workloads | length >  0 and workload.args.load | default('True')```

Same for the run_ycsb var below